whetstone/src/main/java/com/deliveryhero/whetstone/logging/GlobalAndroidComponentListener.kt

```diff
@@ -0,0 +1,15 @@
+package com.deliveryhero.whetstone.logging
+
+import java.lang.IllegalStateException
+
+public object GlobalAndroidComponentListener {
+    public var componentInjectionListener : ComponentInjectionListener? = null
```

Comment:
could probably be replaced by something like 
```
    var componentInjectionListener : ComponentInjectionListener? by Delegates.vetoable(null) { _, oldValue, newValue -> 
        oldValue == null || newValue == null 
    }

```

Reply:
This one seems more readable for someone who know these kotlin function but it does not have the functionality I need and explained in other comment.